pilz_robot_programming/src/pilz_robot_programming/commands.py

```diff
@@ -217,39 +221,58 @@ def _cmd_to_request(self, robot):
         if self._goal is None:
             raise NameError("Goal is not given.")
```

Comment:
Can we get rid of these lines? (EAFP)

Reply:
yes, of course. Just go ahead. Or put it into a follow-up PR, since it is out-of-scope / unchanged in the current PR.

Reply:
Imho it is not out-of-scope. You changed how the member `self._goal` is treated inside this method. We should keep it somehow consistent. Of course we cannot change the coding-style of the whole API...